modules/component/src/let/let.directive.ts

```diff
@@ -148,35 +163,37 @@ export class LetDirective<U> implements OnDestroy {
   }
 
   @Input()
-  set ngrxLet(potentialObservable: ObservableInput<U> | null | undefined) {
-    this.cdAware.nextPotentialObservable(potentialObservable);
+  set ngrxLet(potentialObservable: PotentialObservable<U>) {
+    this.renderEventManager.nextPotentialObservable(potentialObservable);
   }
 
   constructor(
-    cdRef: ChangeDetectorRef,
-    ngZone: NgZone,
+    private readonly cdRef: ChangeDetectorRef,
+    private readonly ngZone: NgZone,
     private readonly templateRef: TemplateRef<LetViewContext<U>>,
     private readonly viewContainerRef: ViewContainerRef,
-    errorHandler: ErrorHandler
-  ) {
-    this.cdAware = createCdAware<U>({
-      render: createRender({ cdRef, ngZone }),
-      resetContextObserver: this.resetContextObserver,
-      updateViewContextObserver: this.updateViewContextObserver,
-      errorHandler,
-    });
-    this.subscription = this.cdAware.subscribe({});
-  }
+    private readonly errorHandler: ErrorHandler
+  ) {}
 
-  createEmbeddedView() {
-    this.isEmbeddedViewCreated = true;
-    this.viewContainerRef.createEmbeddedView(
-      this.templateRef,
-      this.viewContext
+  ngOnInit(): void {
+    this.subscription.add(
+      this.renderEventManager.handlePotentialObservableChanges().subscribe()
```

Comment:
After playing around with the suspense template, I realized that the potential observable subscription has to be moved to the `ngOnInit` method in order to get initial input values on directive initialization.